client/src/components/statistics/SeriesChart.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { BarChart, Bar, XAxis, YAxis, CartesianGrid, Tooltip, Label } from 'recharts';
+
+const SeriesChart = ({ series, dataKey }) => {
+  const data = series.map((s) => ({ name: s.name, [dataKey]: s[dataKey]}));
+  return (
+    <BarChart 
```

Comment:
Add onClick and it'll navigate to relevant series details (tried myself, it should work):

```
const SeriesChart = ({ series, dataKey }) => {
  const navigate = useNavigate();
  const data = series.map((s) => ({ _id: s._id, name: s.name, [dataKey]: s[dataKey]}));

  const barClickHandler = (event) => {
    if (+event?.activePayload?.length > 0) {
      const { _id } = event.activePayload[0].payload;
      navigate(`/series/${_id}`);
    }
  };

  return (
    <BarChart 
      width={800}
      height={300}
      data={data}
      margin={{ top: 20, right: 10, left: 10, bottom: 5 }}
      onClick={barClickHandler}
    >
```
```